src/03_modules.py

```diff
@@ -10,22 +10,28 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+print(sys.argv)
```

Comment:
Print out the command line arguments in sys.argv, one per line.

Probably means you'll want to loop over the list when you see something like "print . . . in . . . one per line."